screens/signup.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Devstore | devlopers mall</title>
+    <link rel="stylesheet" href="../styles/gobal.css" />
+    <link rel="stylesheet" href="../styles/home.css" />
+    <link rel="stylesheet" href="../styles/auth.css" />
```

Comment:
Have one file which you apply to every page and then import these files to that file so there won't be so much clutter in your HTML files.

Reply:
Thanks @dipen2000  will look over this suggestions.